src/openmrs-react-root-decorator.js

```diff
@@ -23,14 +31,17 @@ export default function decorateOptions(opts) {
           // TO-DO have a UX designed for when a catastrophic error occurs
           return null;
         } else {
+          const rootComponent = (
+            <ModuleNameContext.Provider value={opts.moduleName}>
+              <React.Suspense fallback="root loading....">
```

Comment:
Probably best not to show `root loading...` to real users. I'm not sure what the best UX would be (maybe a visual loader of some kind that we could talk to greg about), but I think rendering nothing is superior to rendering the text `root loading....`. Additionally, we don't even know where to position this text/loader since we don't know where this particular single-spa application is going to position itself. So I think rendering nothing might be best.

```suggestion
              <React.Suspense fallback={null}>
```